fml/time/time_point.cc

```diff
@@ -21,14 +21,23 @@ TimePoint TimePoint::Now() {
   return TimePoint(zx_clock_get_monotonic());
 }
 
+TimePoint TimePoint::CurrentWallTime() {
+  return Now();
+}
+
 #else
 
 TimePoint TimePoint::Now() {
   // The base time is arbitrary; use the clock epoch for convenience.
-  const auto elapsed_time = std::chrono::steady_clock::now().time_since_epoch();
+  const auto elapsed = std::chrono::steady_clock::now().time_since_epoch();
```

Comment:
This code could be converted to a template function that would be parameterized on either `steady_clock` or `system_clock`